### PR TITLE
refactor: fixed issues in multimeter

### LINKED ISSUE #1229: Design issues and typos for Multimeter
**Actual Behaviour**
Typo for the word "Capacitance"
Text not justified
Lines overlay the unit display area
Radio button switches colour when tapped

**Expected Behaviour**
No typos, justified text, lines don't overlay, toggle switch button to have consistent colour (remains grey when tapped)

**Screenshots of the issue**

**Would you like to work on the issue?**
@abhinavraj23

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #1229 

**Changes**: 
Fixed typos, solved overlapping of lines,fixed toggling of switch, fixed knob marker issues of knob

**Screenshot/s for the changes**: 
 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]
[fixed_issues_UI.apk.zip](https://github.com/fossasia/pslab-android/files/2196150/fixed_issues_UI.apk.zip)

### CODE REVIEW

**Comment by abhinavraj23** [comment]
@cweitat sorry for those issues, specially for the overlapping issues , it was not there in any of the devices I have tested, but now I have solved all the issues in this PR, I have also provide the apk :+1:

**Comment by abhinavraj23** [comment]
@CloudyPadmal @harsh-2711 @Vikum94 @Avjeet Please review :+1:

**Comment by cweitat** [comment]
Look at this, do you see that the knob is moving up and down when spin? Also, the switch radio button still turns red upon switching. The colour should remain as grey.

ezgif com-video-to-gif

**Comment by CloudyPadmal** [comment]
@abhinavraj23 can we make the layout non-scrollable and the knob and everything resize according to screen size?

**Comment by abhinavraj23** [comment]
@CloudyPadmal No it doesn't fits in every screen without scroll-view, it becomes very diminished in case of small mobiles and doesn't looks good

**Comment by cweitat** [comment]
@abhinavraj23 does shrinking the area where it display the numbers works?

**Comment by cweitat** [comment]
@abhinavraj23 Yes please go ahead. also, change the information at the bottom to the dialog box as shown at #1229

**Comment by abhinavraj23** [comment]
@cweitat I think it's a better idea to have the information in the information-box because having it in the dialog box will often cause problems to the user, as they might often have accidental clicks (due to small space) and end up opening the dialog-box often time

**Comment by abhinavraj23** [comment]
Other than the dialog-box change rest all the other changes are done :+1: . Please review @CloudyPadmal @harsh-2711 @Avjeet @Vikum94

File: app/src/main/res/layout/activity_multimeter.xml [hunk 1]
```diff
@@ -126,7 +126,7 @@
                 android:id="@+id/lower_line"
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/multimeter_line_width"
-                android:layout_marginBottom="@dimen/multimeter_constraint_2"
+                android:layout_marginBottom="20dp"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/activity_multimeter.xml]: If you add these to dimens.xml, you can easily change dimensions from there.

**Comment by cweitat** [comment]
I'm not sure how it looks like on the small screen, maybe you can share with me the screenshot. On my screen, I have so much space for another small information icon.

**Comment by abhinavraj23** [comment]
@cweitat This is the screenshot iof implementation in nexus-one (the smallest android), as you can see that implementing a information tip will not look good and will often lead to accidental clicks, I think the current implementation is fine.

**Comment by abhinavraj23** [comment]
@CloudyPadmal I have done the changes as requested :+1:

**Comment by cweitat** [comment]
@abhinavraj23 if that's the case then I approve the changes. Thanks for the screenshot.

**Comment by abhinavraj23** [comment]
Thanks a lot @cweitat :+1:

**Comment by abhinavraj23** [comment]
@CloudyPadmal @harsh-2711 @Avjeet Please review

**Pull request merged**